chapter_notation/index_vn.md

```diff
@@ -78,14 +91,26 @@ The notation used throughout this book is summarized below.
 * $\prod$: Series multiplication
 -->
 
-*dịch đoạn phía trên*
+* $f(\cdot)$: một hàm số
+* $\log(\cdot)$: logarit tự nhiên
+* $\exp(\cdot)$: hàm $e$ mũ
+* $\mathbf{1}_\mathcal{X}$: The indicator function <!-- Thảo luận -->
```

Comment:
Không biết có sách toán nào dịch từ này chưa, nếu không thì mình đề xuất thế này.
```suggestion
* $\mathbf{1}_\mathcal{X}$: Hàm chỉ định
```

Reply:
```suggestion
* $\mathbf{1}_\mathcal{X}$: hàm đặc trưng (trả về 1 nếu đối số là một phần tử thuộc $\mathcal{X}$, trả về 0 trong trường hợp còn lại).
```

Cụm này có từ đồng nghĩa là `characteristic function` được dịch là `hàm đặc trưng` trong một số tài liệu tiếng Việt: https://vi.wikipedia.org/wiki/T%E1%BA%ADp_m%E1%BB%9D

Hàm này chỉ một phần tử có thuộc một tập hợp hay không. Trả về 1 nếu có, 0 nếu không.